addons/mail/static/src/models/discuss/discuss.js

```diff
@@ -242,6 +228,32 @@ registerModel({
             }
             return this.addingChannelValue;
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeChannelMobileAddItemHeaderInputView() {
+            if (
+                this.messaging.device.isMobile &&
+                this.isAddingChannel
```

Comment:
`isAddingChannel` and `isAddingChat` should be removed in favor if just the existence of the view